frontend/src/bundles/common/components/components.ts

```diff
@@ -1,6 +1,8 @@
 export { Button } from './button/button.js';
 export { Input } from './input/input.js';
 export { Link } from './link/link.js';
+export { Loader } from './loader/loader.js';
+export { LoaderOverlay } from './loader-overlay/loader-overlay.js';
```

Comment:
Please unite these 2 components

Reply:
Or pass Loader through props as children and leave it as Overlay

Reply:
you can export both of them but there is no point in just importing Loader in LoaderOverlay component

Make it accept any component as children

Reply:
Okay, I've just done this.